content/docs/concepts/security/index.md

```diff
@@ -10,197 +10,230 @@ aliases:
     - /docs/concepts/security/rbac/
 ---
 
-Istio aims to enhance the security of microservices and their communication
-without requiring service code changes. It is responsible for:
+Istio offers strong security to the microservices and their communication.
+For example, traffic encryption to defend against the man-in-the-middle attack,
+mutual TLS and fine-grained access policies to provide flexible service access control
+and auditing to detect who did what at what time.
 
-- Providing each service with a strong identity that represents its role to
-  enable interoperability across clusters and clouds
+This page gives an overview of how Istio security can be leveraged to secure your services, wherever you run them.
+In particular, Istio security mitigates both insider and external threats against your data, endpoints, communication and platform.
 
-- Securing service to service communication and end-user to service
-  communication
+{{< image width="80%" ratio="56.25%"
+    link="./overview.svg"
+    alt="Istio security overview."
+    caption="Istio Security Architecture"
+    >}}
+
+Istio security provides strong identity, powerful policy, transparent TLS encryption and AAA (authentication/authorization/audit)
+to protect your services and data. The goals of Istio security are:
+
+- **Security by default**: no changes needed for application code and infrastructure
+
+- **Defense in depth**: integrate with existing security systems to provide multiple layers of defense
 
-- Providing a key management system to automate key and certificate generation,
-  distribution, rotation, and revocation
+- **Zero-trust network**: build security solutions on untrusted networks
 
-The following diagram shows Istio's security architecture, which includes three
-primary components: identity, key management, and communication security. The
-diagram shows how to use Istio to secure the service-to-service communication
-between the `frontend` service running as the `frontend-team` service account
-and the `backend` service running as the `backend-team` service account. Istio
-supports services running on Kubernetes containers, virtual machines, and
-bare-metal machines.
+This guide provides a high-level overview of Istio security; you can find more information about specific Istio security concepts
```

Comment:
We already introduced this document earlier. I would remove lines 36 and 37 to avoid redundancies.